bindings/core/src/method/client.rs

```diff
@@ -369,4 +369,9 @@ pub enum ClientMethod {
         /// The address for request funds
         address: Bech32Address,
     },
+    /// Returns a block ID (Blake2b256 hash of block bytes) from a block
```

Comment:
Outdated comment (also in the utils enum)

Reply:
Suggestions for a better description?

Reply:
Maybe just remove `(Blake2b256 hash of block bytes)` since this is not correct anymore
If that's then not enough information we could link to the TIP https://github.com/iotaledger/tips-draft/blob/tip46/tips/TIP-0046/tip-0046.md#block-id (would need to be adjusted when it's an actual TIP)